ores/scoring_contexts/scoring_context.py

```diff
@@ -70,7 +70,7 @@ def extract_roots(self, model, rev_ids, caches=None):
         root_ds = [d for d in dependencies.dig(features)
                    if isinstance(d, Datasource)]
         error_root_vals = self.extractor.extract(rev_ids, root_ds,
-                                                 cache=caches)
+                                                 caches=caches)
```

Comment:
https://github.com/wiki-ai/revscoring/pull/251

You'll also need to make a release and bump the requirements.txt here.